drydock/patches/kustomization

```diff
@@ -46,3 +46,109 @@ patches:
       value:
         argocd.argoproj.io/sync-wave: "5"
 {%- endif %}
+{% if DRYDOCK_ENABLE_CELERY_TUNING %}
+- target:
+    kind: Deployment
+    name:  "cms-worker"
+  patch: |-
+    - op: add
+      path: /spec/template/spec/containers/0/args
+      value:
+      - celery
+      - --app=cms.celery
+      - worker
+      - --loglevel=info
+      - --concurrency=1
+      - --hostname=edx.cms.core.default.%%h
+      - --max-tasks-per-child=1
+      - --prefetch-multiplier=1
+      - --exclude-queues=edx.lms.core.default
```

Comment:
I think these are better as straticmerge patches? We could them in a different files.